services/distribution/src/main/java/app/coronawarn/server/services/distribution/statistics/keyfigurecard/factory/InfectionsCardFactory.java

```diff
@@ -77,6 +82,11 @@ protected List<Optional<Object>> getRequiredFieldValues(StatisticsJsonStringObje
       return List.of(Optional.empty());
     }
 
+    if (stats.getInfectionsReportedDaily() < stats.getInfectionsReported7daysAvg() * 0.01) {
+      logger.warn("skipping '{}' for '{}', because the reported infections is less than {}% of the last 7 days average", stats.getEffectiveDate(),
+          INFECTIONS_CARD, 0.01);
```

Comment:
```suggestion
      logger.warn("skipping '{}' for '{}', because the reported infections are less than {}% of the last 7 days average", stats.getEffectiveDate(),
          INFECTIONS_CARD, 0.01 * 100);
```